FINAL_COUNTDOWN.md

```diff
@@ -0,0 +1,75 @@
+This is the burn down issue to ship Glimmer 2. While there are still in flight PRs, this is known state of the world at SHA f1416d78c5f2fe3668e807b21d6508a164dac98c.
+
+![Awesome 80s hair band](http://vignette2.wikia.nocookie.net/degrassi/images/d/d4/Final-Countdown.gif)
+
+# Macro Tasks
+- [ ] Complete all test migration described in #13127
+- [ ] Migrate/coalesce fragment tests #13152
+
+# Features
+
+### Infra
+- [ ] Integrate into run-loop
+
+### Helpers
+- [ ] Implement `{{partial}}` [Failing Tests](https://github.com/emberjs/ember.js/blob/f1416d78c5f2fe3668e807b21d6508a164dac98c/packages/ember-glimmer/tests/integration/helpers/partial-test.js)
+- [ ] Implement `{{mut}}`
+- [ ] Implement `{{textarea}}`: [Failing test 1](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L15) [Failing test 2](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L23) [Failing test 3](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L36)
+- [ ] Implement `{{each-in}}` [Failing tests](packages/ember-glimmer/tests/integration/syntax/each-in-test.js#L23)
+- [ ] Implement `{{action}}`
+
+### Currly Components
+- [ ] Implement remaining life-cycle hooks
+- [ ] Implement custom Ids for curly component invocations: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L26)
+- [x] Implement triple curiles #13329
+- [x] Implement `htmlSafe` #13329
```

Comment:
`htmlSafe` is not actually supported in double-curlies from what I can tell. The PR mentioned here only fixes the tests to make it possible to compare HTMLBars and Glimmer2 behavior side-by-side. The feature is still missing (and I'm trying to implement it).